committeeoversightapp/templates/list.html

```diff
@@ -1,29 +1,31 @@
 {% extends 'base.html' %}
 
 {% block content %}
-<div class="col col-lg-11">
+<div class="col">
```

Comment:
Is this a bootstrap class? I thought cols needed some kind of designation, e.g., col-md-12? Asking for my own learning!

Reply:
`.col` will take up all the space it can! https://getbootstrap.com/docs/4.1/layout/grid/#all-breakpoints

Reply:
neat. thanks, bootstrap 4.